src/Cli/OrgCommands.php

```diff
@@ -251,6 +255,71 @@ public function orgUpdateProjectsInfo($csv_file, $options = [
         }
     }
 
+    /**
+     * @command org:update-projects-merge-prs
+     * @param $csv_file The path to csv file that contains projects information.
+     */
+    public function orgUpdateProjectsMergePrs($csv_file, $options = [
+        'as' => 'default',
+        'age' => 30,
+        'pr-title-query' => '[UpdateTool - Project Information]',
+        'pr-title-pattern' => '',
+    ])
+    {
+        $api = $this->api($options['as']);
+        $prTitleQuery = $options['pr-title-query'];
+        $prTitlePattern = $options['pr-title-pattern'];
+        $age = $options['age'];
+
+        if (file_exists($csv_file)) {
```

Comment:
Can we use the GitHub API to find the open PRs? Can we search for PRs across all of the projects in an organization,  and match by query, or do we have to search for PRs in a specific project?

Reply:
It looks like this is available, e.g. https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Apantheon-systems+archived%3Afalse+

Did not find API equivalent, but should be doable. Might need a new hubph wrapper function, but I'd guess it's possible with the knplabs github api.

Reply:
Then we can just forget about the cvs file in this command.